nss-day-cohort-51/holiday-road-grand-canyon-gliders
# Updated PopUp Modals (Park & Events details)

# Updated PopUp Modals (Park & Events details) to include photos and updated styling with spacing, padding, and scrolling descriptions so modal doesn't disappear below screen

## Type of change


# Testing Instructions

Choose park and click on details button to see new photo included on popup modal. Click on Events in the saved trips section to see Events modal popup and make sure either a custom photo shows up or generic NPS stock photo.

# Checklist:
